misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestAnnotationProcessor.java

```diff
@@ -465,7 +372,8 @@ private void validateFunctionName(String functionName, SimpleVariableNode variab
 
             // If it reaches this part, then the function has'nt been found in both packages
             diagnosticLog.logDiagnostic(Diagnostic.Kind.ERROR, attachmentNode.getPosition(),
-                    "Function \'" + functionName + "\' cannot be found in the current package");
+                    "Function \'" + functionName + "\' cannot be found in the package \'"
+                            + functionToMockID.toString());
```

Comment:
`functionToMockID` is nullable. NPE can be thrown.

Reply:
There is a null check for functionToMockID before which throws an error. Woudlnt that be enough or should the NPE be handled here as well.

Reply:
Is it line 351, Is yes, That's not enough because there is no else part for that condition and still NPE can be thrown right?

Reply:
I added a check for the functionToMockID. If it doesnt exist, then it will show up as a compile-time error